src/App.scss

```diff
@@ -59,6 +59,14 @@ code {
   overflow: auto;
 }
 
+.Announcement {
+    background-color: var(--color-note);
+    color: var(--color-text-contrast);
+    text-align: center;
+    width: 100%;
+    margin-bottom: 1rem;
+}
```

Comment:
To make the link the same color as the text, we need to add this inside the `.Announcement` class:

```css
.Announcement {
  ...
  padding: 1rem 0; // We might add vertical padding to let the text breathe a bit

  // This is the class of the anchor
  .TextLink {
    color: inherit;
  }
}
```

Spacing looks a bit off here (we use 2 spaces everywhere).

Reply:
@quietbits spacing on the padding?